src/useField.js

```diff
@@ -161,14 +161,14 @@ function useField<FormValues: FormValuesShape>(
     get checked() {
       let value = state.value;
```

Comment:
value should probably be `const` as you can call `parse` right here, then you could remove line 164 and change 171 to `return value === _value"